CMakeLists.txt

```diff
@@ -782,6 +782,9 @@ if(USE_COREML_DELEGATE)
   string(APPEND CMAKE_CXX_FLAGS " -DUSE_COREML_DELEGATE")
 endif()
 
+# # Lets try adding cutlass manually
+# string(APPEND CMAKE_CXX_FLAGS " -DUSE_CUTLASS")
+
```

Comment:
This was cannon fodder.. but something like this flag for `-DUSE_FLASH_ATTENTION` would probably be useful if it was wired up correctly